src/sentry/notifications/platform/slack/provider.py

```diff
@@ -13,18 +24,37 @@
 )
 from sentry.organizations.services.organization.model import RpcOrganizationSummary
 
-# TODO(ecosystem): Figure out a way to use 'SlackBlock' type
-type SlackRenderable = Any
+
+class SlackRenderable(TypedDict):
+    blocks: list[Block]
 
 
 class SlackRenderer(NotificationRenderer[SlackRenderable]):
-    provider_key = NotificationProviderKey.DISCORD
+    provider_key = NotificationProviderKey.SLACK
 
     @classmethod
     def render[DataT: NotificationData](
         cls, *, data: DataT, rendered_template: NotificationRenderedTemplate
     ) -> SlackRenderable:
-        return {}
+        subject = HeaderBlock(text=PlainTextObject(text=rendered_template.subject))
```

Comment:
🤔 should there be a generic implementation of this in the [NotificationRenderer](https://github.com/getsentry/sentry/blob/a9043737ea8ae05e284b624c41d81b0e2b6f1443/src/sentry/notifications/platform/renderer.py#L22-L34)?

That renderer could just check for the different areas of a `rendered_template` (header, body, footer) and farm into individual Renderers to do any custom rendering for individual sections.